twine/repository.py

```diff
@@ -28,6 +28,9 @@
 
 KEYWORDS_TO_NOT_FLATTEN = set(["gpg_signature", "content"])
 
+LEGACY_PYPI = 'https://pypi.python.org/pypi/'
```

Comment:
I think you want this to just be ``https://pypi.python.org/`` and move the ``pypi/`` into the string format. The upload URL is typically ``https://pypi.python.org/pypi`` (note the missing slash at the end).